next-packages/is-prop-valid/src/props.js

```diff
@@ -454,10 +457,12 @@ const props = {
   yChannelSelector: true,
   z: true,
   zoomAndPan: true,
+  // preact
   for: true,
-  class: true
+  class: true,
+  autofocus: true
```

Comment:
`autofocus` is unnecessary as I made the regex case insensitive and `autoFocus` is already there